<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>523</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>902</width>
    <height>523</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>902</width>
    <height>523</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Heimdall Frontend 1.1</string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="statusTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="accessibleName">
   <string notr="true"/>
  </property>
  <property name="accessibleDescription">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string notr="true"/>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>MS Shell Dlg 2</family>
    </font>
   </property>
   <widget class="QTabWidget" name="functionTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>881</width>
      <height>491</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="flashTab">
     <attribute name="title">
      <string>Flash</string>
     </attribute>
     <widget class="QGroupBox" name="pdaGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>421</width>
        <height>331</height>
       </rect>
      </property>
      <property name="title">
       <string>PDA/Code</string>
      </property>
      <widget class="QGroupBox" name="factoryfsGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>FactoryFS</string>
       </property>
       <widget class="QLineEdit" name="factoryfsLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="factoryfsBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="factoryfsCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="kernelGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Kernel (zImage)</string>
       </property>
       <widget class="QLineEdit" name="kernelLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="kernelBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="kernelCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="primaryBootGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Primary Bootloader</string>
       </property>
       <widget class="QLineEdit" name="primaryBootLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="primaryBootBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="primaryBootCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="secondaryBootGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Secondary Bootloader</string>
       </property>
       <widget class="QLineEdit" name="secondaryBootLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="secondaryBootBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="secondaryBootCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="paramGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Param.lfs</string>
       </property>
       <widget class="QLineEdit" name="paramLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="paramBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="paramCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="cscGroupBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>421</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>CSC</string>
      </property>
      <widget class="QGroupBox" name="cacheGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Cache</string>
       </property>
       <widget class="QLineEdit" name="cacheLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="cacheBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="cacheCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="databaseGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Database Data</string>
       </property>
       <widget class="QLineEdit" name="databaseLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="databaseBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="databaseCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="repartitionGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Repartition</string>
      </property>
      <widget class="QGroupBox" name="pitGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>401</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>PIT</string>
       </property>
       <widget class="QLineEdit" name="pitLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>301</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="pitBrowseButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>50</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="repartitionCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>381</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Repartition Device</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="startFlashButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>780</x>
        <y>410</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="flashProgressBar">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>410</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
     </widget>
     <widget class="QGroupBox" name="otherGroupBox">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>160</y>
        <width>421</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
       </font>
      </property>
      <property name="title">
       <string>Other</string>
      </property>
      <widget class="QGroupBox" name="recoveryGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Recovery</string>
       </property>
       <widget class="QLineEdit" name="recoveryLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="recoveryBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="recoveryCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="modemGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>401</width>
         <height>61</height>
        </rect>
       </property>
       <property name="title">
        <string>Modem</string>
       </property>
       <widget class="QLineEdit" name="modemLineEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>281</width>
          <height>21</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QPushButton" name="modemBrowseButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>71</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="modemCheckBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLabel" name="flashLabel">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>380</y>
        <width>411</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string notr="true">Ready</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="firmwareTab">
     <attribute name="title">
      <string>Download Firmware</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>851</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>831</width>
         <height>391</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string>This functionality is still under development and will be included in a future release.</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>902</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutHeimdall"/>
   </widget>
   <widget class="QMenu" name="menuDonate">
    <property name="title">
     <string>Donate</string>
    </property>
    <addaction name="actionDonate"/>
   </widget>
   <addaction name="menuDonate"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAboutHeimdall">
   <property name="text">
    <string>About Heimdall</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate to Glass Echidna</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
